lib/rbs/prototype/rb.rb

```diff
@@ -686,6 +694,12 @@ def range_element_type(types)
 
       def param_type(node, default: Types::Bases::Any.new(location: nil))
         case node.type
+        when :INTEGER
+          BuiltinNames::Integer.instance_type
+        when :FLOAT
+          BuiltinNames::Float.instance_type
+        when :RATIONAL, :IMAGINARY
+          BuiltinNames::Numeric.instance_type
```

Comment:
How about to like this?

```suggestion
        when :RATIONAL
          Types::ClassInstance.new(name: TypeName("::Rational"), args: [], location: nil)
        when :IMAGINARY
          Types::ClassInstance.new(name: TypeName("::Complex"), args: [], location: nil)
```

Reply:
Thanks for the review!
I've fixed it in the following commit.
https://github.com/ruby/rbs/pull/1711/commits/225ed2205df26f51cb4f7f24139ba34c8a1a0fcd